Dockerfile.imagery

```diff
@@ -0,0 +1,27 @@
+FROM osgeo/gdal:ubuntu-small-3.6.1
+
+RUN apt-get update
+# Install pip
+RUN apt-get install python3-pip -y
+# Install Poetry
+RUN pip install poetry
+
+# Define the working directory for the following commands
+WORKDIR /app
+
+# Add Poetry config
+COPY poetry.lock pyproject.toml /app/
+
+# Install Python dependencies
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction --no-ansi --with imagery --without dev
```

Comment:
`--with imagery` will install the `imagery` optional dependencies specified in `pyproject.toml`